hack/generate-standard-manifests.sh

```diff
@@ -121,11 +121,21 @@ VALUES_FILES=$(cd $VALUES_DIR && find * -type f -name "*.yml" )
 # Suppress potential Helm warnings about invalid permissions for Kubeconfig file
 # by throwing away related warnings.
 for values in $VALUES_FILES; do
-    $HELM template \
+    $HELM template --include-crds \
           --namespace kube-system \
           -f "$VALUES_DIR/$values" \
           $EXTRA_VALUES \
           "$ANTREA_CHART" \
           > "$OUTPUT_DIR/$values" \
           2> >(grep -v 'This is insecure' >&2)
 done
+
+# We also generate a manifest which only includes CRD resources (all of them).
+# This is useful for Antrea upgrades when using Helm.
+CRD_FILES=$(cd $ANTREA_CHART/crds && find * -type f -name "*.yaml" )
+CRD_OUTPUT_FILE="$OUTPUT_DIR/antrea-crds.yml"
+touch "$CRD_OUTPUT_FILE"
+for crd in $CRD_FILES; do
+    echo "---" >> "$CRD_OUTPUT_FILE"
+    cat "$ANTREA_CHART/crds/$crd" >> "$CRD_OUTPUT_FILE"
```

Comment:
It seems generating CRD is not idempotent, executing the script twice will append duplicate content while generating the other manifest won't. Since we normally just call `make manifest` after making a change to any manifest, it would be better to be idempotent?

Reply:
thanks, I didn't think of this. I'll sort the files to make it idempotent.

Reply:
Oh I see now, it's not a sorting issue, it's just that I don't truncate the file :)